src/playercards/customizable/SummonedServitorUpgradeSheet.ttslua

```diff
@@ -35,32 +35,30 @@ inputBoxes = {}
 local SLOT_ICON_POSITIONS = {
     arcane = { x = 0.160, z = 0.65},
     ally = { x = -0.073, z = 0.65}
-  }
-  
-local selectedSlot = ""
+}
+
+-- These match with ArkhamDB's way of storing the data in the dropdown menu
+local slotIndices = {arcane = "1", ally = "0", none = ""}
+
+local selectedSlot = slotIndices.none
 
 -- override 'marked boxes' for debugging ('all' or 'none')
 markDEBUG = ""
 
 -- save state when going into bags / decks
 function onDestroy() self.script_state = onSave() end
 
-function onSave() return JSON.encode({ markedBoxes, inputValues, selectedSlot }) end
+function onSave() return JSON.encode({ markedBoxes, {selectedSlot} }) end
```

Comment:
This shouldn't actually affect the checkboxes at all.  It was actually a mistake for me to make the save state to be 3 items long at all, since the way it works is the first item stores checkbox data, and the second item stores all non-checkbox data.  This is the inputValues variable for all of the textbox customizables (eg. Grizzled), but is not used in the other non-checkbox, non-textbox input card, Living Ink (which is largely what I modelled this one against).

Any references to inputValues are vestigial references to textboxes that are simply not present in Living Ink or Summoned Servitor